src/dsymbol.d

```diff
@@ -166,12 +166,16 @@ alias PASSinline = PASS.PASSinline;
 alias PASSinlinedone = PASS.PASSinlinedone;
 alias PASSobj = PASS.PASSobj;
 
+// Search options
 enum : int
 {
     IgnoreNone              = 0x00, // default
     IgnorePrivateMembers    = 0x01, // don't find private members
     IgnoreErrors            = 0x02, // don't give error messages
     IgnoreAmbiguous         = 0x04, // return NULL if ambiguous
+    IgnoreImports           = 0x08, // don't find symbols in imports
+    IgnoreThisModule        = 0x10, // only look in imports
+    IgnoreRoot              = 0x20, // this search is for a root module
```

Comment:
> I don't think that's an excuse

I am not a fan of making comprehensive overarching PRs. Having smaller, targeted ones make them easier to review and much better focused. Refactorings in particular should be in separate PRs and reviewed on their own merits. Many PRs grow stale because they do way too much, hence nobody wants to pull them, and the poor submitter gets stuck with regularly rebasing them.

Renaming the enum is a refactoring.

This PR is already very large. I would have done it in smaller chunks if I could have.